File: perl/FizzBuzz.pl
#実行方法
#コンソールにてプログラムがあるディレクトリに移動する
#コンソールにて" perl FizzBuzz.pl"を実行する

my $i;

for ($i = 1; $i < 100; $i++){
  if($i % 3 == 0 && $i % 5 == 0){
    print "FizzBuzz\n";
  }elsif($i % 3 == 0){
      print "Fizz\n";
  }elsif($i % 5 == 0){
      print"Buzz\n";
  }else{
      print"$i\n";
  }
}

File: Ruby/FizzBuzz.rb
#
#実行方法
#コンソールにてプログラムがあるディレクトリに移動する
#コンソールにて"ruby FizzBuzz.rb"を実行する
#

for i in 1..100 do
    if i % 3 == 0 && i % 5 == 0
        #改行あり出力printだと改行なし
        puts("FizzBuzz")
    elsif i % 3 == 0
        puts("Fizz")
    elsif i % 5 == 0
        puts("Buzz")
    else
        puts(i)
    end 
end